README.md

```diff
@@ -1,8 +1,8 @@
-# aws-v4-signer-java
+# aws-v4-signer
```

Comment:
This readme needs more work and will be changed once we do the release and merge everuthing back to master.